Automattic/sensei
# Send email to teacher when a student completes a course

Resolves https://github.com/Automattic/sensei/issues/6478

## Proposed Changes

* Allow adding patterns as email template's content.
* Added relevant code and pattern for Sending teacher email on student's course completion
* Disabled the legacy version of this email

## Testing Instructions

- Enable the `sensei_email` post type
- Go to Sensei LMS -> Tools and run the **Re-create Emails** tool
- Go to Sensei LMS -> Settings -> Teacher Emails
- Make sure the **[student:displayname] completed [course:name]** email is there
- Now Create a course with one or more lessons
- Take the course as a student and complete it
- Check that the Course Teacher has got the intended email

## Pre-Merge Checklist
